tests/09_storable_functions.ipynb

```diff
@@ -235,11 +237,14 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "2020-11-09 14:12:39,320 - openpathsampling.experimental.simstore.sql_backend - INFO - Registering storable function: UUID: 80100521955429081778427985284781047820 (float)\n"
+      "2021-07-22 12:05:35,881 - openpathsampling.experimental.simstore.sql_backend - INFO - Registering storable function: UUID: 218455970078904451893013535049010642956 (float)\n"
      ]
     }
    ],
    "source": [
+    "from openpathsampling.experimental.simstore.attribute_handlers import StandardHandler\n",
+    "backend.serialization['float'] = StandardHandler('float')\n",
+    "\n",
```

Comment:
Is this the only code change (it was the only one I could spot in-between all the output changes), if so, this LGTM

Reply:
Yep, that's it. This is done internally in `StorageFunctionHandler.register_function`, although I'm not sure that's the best place for it, in terms of level of abstraction. It seems like maybe this should be handled within the backend, e.g., as part of `backend.register_storable_function`. But I think there was some reason I couldn't do that (or maybe doing that would make other things messy).

https://github.com/openpathsampling/openpathsampling/blob/8eca3946ce68ea44b31709ddb8112812fe24fd79/openpathsampling/experimental/simstore/storable_functions.py#L568-L573